shared_utils/key_pattern_manager.py

```
@@ -89,30 +89,29 @@ def is_o_family_for_openai(llm_model):
     avail_key_list = []
     key_list = keys.split(',')
+    model_lower = llm_model.lower()
 
-    if llm_model.startswith('gpt-') or llm_model.startswith('chatgpt-') or \
-       llm_model.startswith('aioagi-') or is_o_family_for_openai(llm_model):
+    if llm_model.startswith('openrouter-'):
         for k in key_list:
-            if is_openai_api_key(k): avail_key_list.append(k)
-
-    if llm_model.startswith('api2d-'):
+            if is_openroute_api_key(k): avail_key_list.append(k)
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)